application/cmd/requester/utils_test.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2020 Insolar Network Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/insolar/insolar/insolar/secrets"
+)
+
+const (
+	binLocation                string = "./../../../bin/requester"
```

Comment:
don't test binary, test code

Reply:
it's not able to read cobra output. I already tried this way. I decided to run cms.Execute() in tests. It works correctly but not able to catch the output

Reply:
you can read cobra's output if set `SetOutput` on command (parent  one for example)

Reply:
but I need a requester output also